components/common/BlogNav.tsx

```diff
@@ -15,15 +15,16 @@ import LoadingLottie from './ui/LoadingLottie';
 const BlogNav = () => {
   const { team } = useParams();
 
-  const res = useGetNavList(team);
+  const res = useGetBlogHeaderInfo(team);
 
   if (!res) return <LoadingLottie width={10} height={10} fit />;
 
-  const { data: navList } = res;
+  const { data: data } = res;
```

Comment:
오잇! 요거 data를 data로 받아오고 있는 것 같은뎅?
```suggestion
const { data: {navList}} = res;
```
이렇게 하면 고냥 아래 navList로 쓸 수 있겠다!